src/classes/groundplaces.ts

```diff
@@ -15,6 +20,14 @@ import {
  * @description GroundPlaces business logic.
  */
 export class GroundPlaces {
+  private readonly Generator: Generator = new Generator();
```

Comment:
could you create the instance in the constructor like Storage ?

Reply:
Done